packages/icons/icons.json

```
@@ -471,280 +471,280 @@
     },
     {
         "displayName": "Logout",
         "iconName": "log-out",
-        "tags": "arrow, leave",
+        "tags": "arrow, leave, next, share, open in new tab, exit",
         "group": "action",
         "codepoint": 62004
     },
     {
         "displayName": "Star",
         "iconName": "star",
-        "tags": "shape, pin, mark, pro",
+        "tags": "shape, pin, mark, pro, new, favorite",
         "group": "interface",
         "codepoint": 62157
     },
     {
         "displayName": "Star: empty",
         "iconName": "star-empty",
-        "tags": "shape, unpin, mark",
+        "tags": "shape, unpin, mark, star, favorite, like, empty star, goal, accomplishment",
         "group": "interface",
         "codepoint": 62156
     },
     {
         "displayName": "Sort: alphabetical",
         "iconName": "sort-alphabetical",
-        "tags": "ascending, array, arrange",
+        "tags": "ascending, array, arrange, sort, alphabetize, sort ascending, sort descending, string sort, sort alphabetically",
         "group": "action",
         "codepoint": 62146
     },
     {
         "displayName": "Sort: numerical",
         "iconName": "sort-numerical",
-        "tags": "ascending, array, arrange",
+        "tags": "ascending, array, arrange, asc, sort, numbers, dropdown, range",
         "group": "action",
         "codepoint": 62150
     },
     {
         "displayName": "Sort",
         "iconName": "sort",
-        "tags": "ascending, array, arrange",
+        "tags": "ascending, array, arrange, descending list, sort list, scroll down, menu, scroll, hamburger, descending, decrease, justified",
         "group": "action",
         "codepoint": 62151
     },
     {
         "displayName": "Folder: opened",
         "iconName": "folder-open",
-        "tags": "file, portfolio, case",
+        "tags": "file, portfolio, case, files, folder",
         "group": "file",
         "codepoint": 61882
     },
     {
         "displayName": "Folder: closed",
         "iconName": "folder-close",
-        "tags": "file, portfolio, case",
+        "tags": "file, portfolio, case, folder, directory, files",
         "group": "file",
         "codepoint": 61880
     },
     {
         "displayName": "Folder: shared",
         "iconName": "folder-shared",
-        "tags": "file, portfolio, case",
+        "tags": "file, portfolio, case, email, upload, save, export, copy, folder",
         "group": "file",
         "codepoint": 61884
     },
     {
         "displayName": "Caret: up",
         "iconName": "caret-up",
-        "tags": "direction, order, up",
+        "tags": "direction, order, up, chevron, arrow, back, pointer left, position marker",
         "group": "interface",
         "codepoint": 61763
     },
     {
         "displayName": "Caret: right",
         "iconName": "caret-right",
-        "tags": "direction, order, right",
+        "tags": "direction, order, right, play, run, execute, triangle, arrow, arrow right, triangle right",
```

I don't want to conflate this with "play"

```suggestion
        "tags": "direction, order, right, triangle, arrow",
```